order/views.py

```diff
@@ -1 +1,76 @@
-from django.shortcuts import render
\ No newline at end of file
+import json
+
+from django.views      import View
+from django.http       import JsonResponse
+
+from order.models      import Cart, Order
+from django.db.models  import F
+
+from user.utils        import check_user
+
+class CartView(View):
+    @check_user
+    def get(self, request):
+        try:
+            user  = request.user
+            order = Order.objects.prefetch_related('carts').get(user=user, order_status_id = 2)
```

Comment:
`CartView`에서 `Cart Model class` 부터 접근해서 구현해주세요~

```suggestion
            order = Order.objects.prefetch_related('carts').get(user=user, order_status_id = 2)
```

Reply:
넵!